lib/src/message_listview.dart

```diff
@@ -138,6 +140,8 @@ class _MessageListViewState extends State<MessageListView> {
             child: Stack(
               alignment: AlignmentDirectional.topCenter,
               children: [
+                if (itemCount == 0 && widget.messageListHeaderBuilder != null)
+                  widget.messageListHeaderBuilder(),
```

Comment:
sorry i think you've explained this but I've forgotten - why can't this go here regardless of if there are any items in the list?

Reply:
@lisamartin00 It's because it's in a `Stack`, and if it's present along with messages, the widget will sit behind the message list and show through.